example/src/screens/Examples/AwareScrollView/index.tsx

```diff
@@ -81,6 +119,17 @@ export default function AwareScrollView({ navigation }: Props) {
             }}
           />
         </View>
+
+        <View style={styles.switchContainer}>
+          <Text>Toggle snapToOffsets</Text>
+          <Switch
+            value={snapToOffsetsEnabled}
+            testID="bottom_sheet_toggle_enabled_state"
```

Comment:
Please, change this `testId` (I think you copied/pasted it) - it causes e2e test to fail, because we have two identical ids and we expect it'll be only one 😓